$MirOS: ports/print/ghostscript/esp/patches/patch-src_unix-end_mak,v 1.1.7.1 2005/03/18 15:51:03 tg Exp $
--- src/unix-end.mak.orig	Wed Jan 19 04:08:19 2005
+++ src/unix-end.mak	Thu Apr 27 16:14:40 2006
@@ -33,7 +33,7 @@ PGDIRS: STDDIRS
 	@if test ! -d $(PSGENDIR)/$(PGRELDIR); then mkdir $(PSGENDIR)/$(PGRELDIR); fi
 	@if test ! -d $(PSOBJDIR)/$(PGRELDIR); then mkdir $(PSOBJDIR)/$(PGRELDIR); fi
 
-PGDEFS=GENOPT='-DPROFILE' CFLAGS='$(CFLAGS_PROFILE) $(GCFLAGS) $(XCFLAGS)'\
+PGDEFS=GENOPT='-DPROFILE' CFLAGS='$(CFLAGS)'\
  LDFLAGS='$(XLDFLAGS) -pg' XLIBS='Xt SM ICE Xext X11' CC_LEAF="$(CC_LEAF_PG)"\
  BINDIR=$(BINDIR)/$(PGRELDIR)\
  GLGENDIR=$(GLGENDIR)/$(PGRELDIR) GLOBJDIR=$(GLOBJDIR)/$(PGRELDIR)\
@@ -53,7 +53,7 @@ DEBUGDIRS: STDDIRS
 	@if test ! -d $(PSGENDIR)/$(DEBUGRELDIR); then mkdir $(PSGENDIR)/$(DEBUGRELDIR); fi
 	@if test ! -d $(PSOBJDIR)/$(DEBUGRELDIR); then mkdir $(PSOBJDIR)/$(DEBUGRELDIR); fi
 
-DEBUGDEFS=GENOPT='-DDEBUG' CFLAGS='$(CFLAGS_DEBUG) $(GCFLAGS) $(XCFLAGS)'\
+DEBUGDEFS=GENOPT='-DDEBUG' CFLAGS='$(CFLAGS)'\
  BINDIR=$(BINDIR)/$(DEBUGRELDIR)\
  GLGENDIR=$(GLGENDIR)/$(DEBUGRELDIR) GLOBJDIR=$(GLOBJDIR)/$(DEBUGRELDIR)\
  PSGENDIR=$(PSGENDIR)/$(DEBUGRELDIR) PSOBJDIR=$(PSOBJDIR)/$(DEBUGRELDIR)
